.github/workflows/main.yml

```diff
@@ -246,10 +285,13 @@
     - "argo-workflows"
     - "calico"
     - "cert-manager"
+    - "cluster-api"
+    - "cluster-api-provider-aws"
     - "cnrm"
     - "consul"
     - "crossplane"
     - "eck-operator"
+    - "external-secrets"
```

Comment:
I forgot this in a previous PR, also trying to fix in #103, which ever merges first the second should rebase.